lms/djangoapps/certificates/views/webview.py

```diff
@@ -631,12 +631,18 @@ def _get_catalog_data_for_course(course_key):
 
     catalog_data = {}
     course_run_fields = []
+    log.info("language specific template is:{flag} for the course:{course_key}".
+             format(flag=course_certificate_settings.language_specific_templates_enabled, course_key=course_key))
     if course_certificate_settings.language_specific_templates_enabled:
         course_run_fields.append('content_language')
+    log.info("include hours of effort is:{flag} for the course:{course_key}".format(
+        flag=course_certificate_settings.include_hours_of_effort, course_key=course_key))
     if course_certificate_settings.include_hours_of_effort:
         course_run_fields.extend(['weeks_to_complete', 'max_effort'])
 
```

Comment:
Can't you combine these two logs into one plus add the third one too course_certificate_settings.include_hours_of_effort